spec/Procedures.tex

```diff
@@ -1295,38 +1295,46 @@ \subsection{Determining More Specific Functions}
  Otherwise neither mapping is more specific.
 \end{itemize}
 
+\subsection{Determining Best Function}
+\label{Determining_Best_Function}
+
+Given two functions $F_1$ and $F_2$, the better function is
+determined by the following steps:
+
+\begin{itemize}
+\item If $F_1$ is more specific than $F_2$, then $F_1$ is a better match.
+\item If $F_2$ is more specific than $F_1$, then $F_2$ is a better match.
```

Comment:
I think it is redundant, but only if if selecting the "best" function only occurs in the flow described above. In particular, somebody just scanning the document might look at the section on selecting the "best" function and ignore the overview. Then they would get the right idea if we leave the redundant information in place.